test/logflare_web/controllers/api/team_controller_test.exs

```diff
@@ -201,4 +203,100 @@ defmodule LogflareWeb.Api.TeamControllerTest do
              |> assert_schema("NotFoundResponse") == %{error: "Not Found"}
     end
   end
+
+  describe "add_member/2" do
+    test "adds an existing user to a team", %{
+      conn: conn,
+      user: user,
+      main_team: main_team
+    } do
+      new_member = insert(:user)
+
+      assert conn
+             |> add_access_token(user, "private")
+             |> post(~p"/api/teams/#{main_team.token}/members", %{email: new_member.email})
+             |> response(204)
+             |> assert_schema("AcceptedResponse") == ""
+
+      team_users = TeamUsers.list_team_users_by(team_id: main_team.id)
+
+      assert Enum.any?(team_users, fn tu ->
+               tu.email == String.downcase(new_member.email)
+             end)
+    end
+
+    test "creates a new team member when adding with non-existent email", %{
+      conn: conn,
+      user: user,
+      main_team: main_team
+    } do
+      new_email = "newuser@example.com"
+
+      assert conn
+             |> add_access_token(user, "private")
+             |> post(~p"/api/teams/#{main_team.token}/members", %{email: new_email})
+             |> response(204)
+             |> assert_schema("AcceptedResponse") == ""
+
+      team_users = TeamUsers.list_team_users_by(team_id: main_team.id)
+      assert Enum.any?(team_users, fn tu -> tu.email == String.downcase(new_email) end)
+    end
+
+    test "returns not found if doesn't own the team", %{conn: conn, main_team: main_team} do
+      invalid_user = insert(:user)
+      new_member = insert(:user)
+
+      assert conn
+             |> add_access_token(invalid_user, "private")
+             |> post(~p"/api/teams/#{main_team.token}/members", %{email: new_member.email})
+             |> json_response(404)
+             |> assert_schema("NotFoundResponse") == %{error: "Not Found"}
+    end
+  end
+
+  describe "remove_member/2" do
```

Comment:
```suggestion
  describe "remove_member/2 for team_user with user record" do
```

Also needs a test for team_user with when there is no user record, which is when they are invited without having a prior account.